cfgov/v1/jinja2/v1/landing-page-2/_template-prototype-text-intro-content.html

```diff
@@ -0,0 +1,7 @@
+<h1 class='m-text-introduction_heading'>Text Introduction</h1>
+
+<h2 class='m-text-introduction_subheading'>Subheading Lorem Ipsum</h2>
```

Comment:
According to content, this is more of a intro paragraph than a subheading and `should be two to three lines on desktop and serve as an introduction to content below`. If it's that long, it might not make sense to make it a `p` with h2 styles.